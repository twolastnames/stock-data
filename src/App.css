
body {
  background-color: #282c34;
  color: white;
  padding: 40px;
  font-family: Arial;
  text-align: center;
}

.date-text {
  stroke: white;
  writing-mode: tb;
  stroke-width: 1;
}

.axis {
  stroke: white;
  stroke-width: 2;
}


