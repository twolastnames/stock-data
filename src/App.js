import React from 'react';
import ReactDom from 'react-dom';
import './App.css';
import SymbolInput from './SymbolInput'
import ResultTable from './ResultTable'

function App() {

  const onSymbol = (symbol, company) => {
    ReactDom.render(<ResultTable company={company} symbol={symbol}/>, document.getElementById('result-table'))
  }
  return (
    <div>
      <SymbolInput symbolListener={onSymbol}/>
      <div id='result-table'/>
    </div>
  );
}

export default App;
