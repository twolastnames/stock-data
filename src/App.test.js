import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders initial state', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot()
});

