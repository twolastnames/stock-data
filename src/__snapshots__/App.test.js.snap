// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders initial state 1`] = `
<div>
  <form>
    <label>
      Stock Symbol
    </label>
    <input
      name="symbol"
      type="text"
    />
    <input
      onClick={[Function]}
      type="button"
      value="Submit"
    />
  </form>
  <div
    id="result-table"
  />
</div>
`;
